src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -5996,6 +5996,32 @@ inline void compute_sum_24_host(__m256d *p, __m256d *pSumR, __m256d *pSumG, __m2
     pSumB[0] = _mm256_add_pd(_mm256_add_pd(p[4], p[5]), pSumB[0]); //add 8B values and bring it down to 4
 }
 
+inline void compute_var_8_host(__m256d *p1, __m256d *pMean, __m256d *pVar)
+{
+    pVar[0] = _mm256_add_pd(_mm256_mul_pd(_mm256_sub_pd(pMean[0], p1[0]), _mm256_sub_pd(pMean[0], p1[0])), pVar[0]);
+    pVar[0] = _mm256_add_pd(_mm256_mul_pd(_mm256_sub_pd(pMean[0], p1[1]), _mm256_sub_pd(pMean[0], p1[1])), pVar[0]);
+}
+
+inline void compute_varchannel_24_host(__m256d *p1, __m256d *pMeanR, __m256d *pMeanG, __m256d *pMeanB, __m256d *pVarR, __m256d *pVarG, __m256d *pVarB)
```

Comment:
This is for 3 channels together

Reply:
Changed "compute_varchannel_24_host" into "compute_variance_pkd3_24_host"

compute_varRGB_24_host uses image mean and it will be confusing to use compute_variance_pln3_24_host or any other naming